repos/fdbt-reference-data-service/src/uploaders/txc_uploader/main.py

```diff
@@ -39,6 +39,12 @@ def lambda_handler(event, context):
             s3_client, cloudwatch_client, bucket, key, file_path, db_connection, logger
         )
     except Exception as e:
+        ssm_client.put_parameter(
```

Comment:
As this lambda is invoked for every file uploaded to S3 - a system wide issue will flood SSM api. The `checkReferenceDataImportHasCompleted` function in table renamer will be able to identify any system wide issues impacting this lambda